roles/wordpress-install/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   known_hosts:
     name: "{{ item.name }}"
     key: "{{ item.key | default(omit) }}"
-    path: "{{ item.path | default('/home/' + ansible_user + '/.ssh/known_hosts') }}"
+    path: "{{ item.path | default(home_root + ansible_user + '/.ssh/known_hosts') }}"
     state: "{{ item.state | default('present') }}"
   with_items: "{{ known_hosts | default([]) }}"
```

Comment:
If you go with `default(omit)` for `path`, let's add `become: no` right above `with_items` (this task only). This will make the task run as `vagrant` (vs. `root`), adding to `known_hosts` in $HOME for `vagrant`. This is good because the subsequent composer install task that needs the `known_hosts` will run as `vagrant` (also uses [`become: no`](https://github.com/roots/trellis/blob/a7506ed3fcf4152da0a01d0e34efa100aedd6033/roles/wordpress-install/tasks/main.yml#L41)).